doc/release-notes-empty-template.md

```diff
@@ -43,7 +43,7 @@ Compatibility
 ==============
 
 Bitcoin Core is supported and tested on operating systems using the
-Linux Kernel 3.17+, macOS 13.0+, and Windows 10 and newer. Bitcoin
+Linux Kernel 3.17+, macOS 13+, and Windows 10 and newer. Bitcoin
```

Comment:
nit: While touching this line, we can use "+" or "and newer" consistently without mixing both.